### ISSUE TITLE
Telemetry voice message: Type of value missing

### ISSUE THREAD

**Issue by boomerschnecke** [EdgeTX/edgetx#4468]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Within the current telemetry voice message its "only" mentioned the value and the coresponding unit.
so f.e. 7 Volts, 4 Amps, 399milliAmperhours ect
Missing is to name beforehand the "type of value"

### Describe the solution you'd like

Missing is to name beforehand the "type of value"
f.e. **Voltage**: 7 Volts, **Current**: 4 Amps, **Capacity**: 300 mAh ect
So the type of value is missing currently (**Volltage, Current, Capacity**). (if possible of course for all telemetry items)
I think this would be a highly appreciated feature update

### Describe alternatives you've considered

no alternative

**Comment by 3djc** [comment]
I don't think it should be added, since the units themselves indicate what the sensor is about.

You can easily create those sounds and prefix the announcement by this if you so desire

**Comment by boomerschnecke** [comment]
Some of them yes, but f.e. Sats: its only a number without anything. If you want to distinguish between ReceiverBatVoltage and normal supply Bat Voltage?. I personly would prefere this

**Comment by 3djc** [comment]
Well in your example, both voltage would say ´ voltage ´, so it would not help.

**Comment by boomerschnecke** [comment]
The intention should be not only voltage, but Battery Voltage or Receiver Voltage or Number of Satelites ect
Jeti Telemetry f.e. is able to do a lot of this things

**Comment by 3djc** [comment]
The issue here is that sensors come in all kind of shapes, forms and protocols here, unlike Jeti closed environments. There is no way to know where a voltage sensor comes from
